mdx-models/src/main/java/com/mx/models/profile/Profile.java

```diff
@@ -6,7 +6,7 @@
 import com.mx.models.MdxBase;
 
 public final class Profile extends MdxBase<Profile> {
-  @SerializedName("birth_date")
+  @SerializedName("birth_date_on")
```

Comment:
Add a fallback for backward compatibility? 
@SerializedName(value="birth_date_on", alternate={"birth_date"})

Reply:
I don't believe this is in use because Synchronicity is coded for "birth_date_on", and does not return a value if the Profile object has "birth_date" set.